populate/src/main/resources/seed/i18n/dev/languageTexts.json

```diff
@@ -101,5 +101,39 @@
   {"keyName": "pageNotFoundMessage", "text": "DEV_If you believe this is an error, please contact ${studyContactEmail}", "language": "dev"},
   {"keyName": "pageNotFoundReturnHome", "text": "DEV_Return to the home page", "language": "dev"},
   {"keyName": "authErrorPageTitle", "text": "DEV_Something went wrong", "language": "dev"},
-  {"keyName": "authErrorPageMessage", "text": "DEV_There was an issue with our authentication service. Please try again. If the problem persists, please contact ${studyContactEmail}.", "language": "dev"}
+  {"keyName": "authErrorPageMessage", "text": "DEV_There was an issue with our authentication service. Please try again. If the problem persists, please contact ${studyContactEmail}.", "language": "dev"},
+  {"keyName": "navbarSampleKits", "text":  "DEV_Sample Kits", "language": "dev"},
+  {"keyName": "kitsPageTitle", "text": "DEV_Sample collection kits", "language": "dev"},
+  {"keyName": "kitsPageDescription", "text": "DEV_Sample collection kits are a valuable part of the study process and can help provide researchers with important insights. Below you will find the status of all kits that have been provided to you.", "language": "dev"},
+  {"keyName": "kitsPageInPersonTitle", "text": "DEV_Provide a sample in-person", "language": "dev"},
+  {"keyName": "kitsPageInPersonDescription", "text": "DEV_This study is currently offering the option to complete a sample collection kit in-person.", "language": "dev"},
+  {"keyName": "kitsPageInPersonCompleteButton", "text": "DEV_Complete a kit in-person", "language": "dev"},
+  {"keyName": "kitsPageYourKitsTitle", "text": "DEV_Your kits", "language": "dev"},
+  {"keyName": "kitsPageStatusPreparing", "text": "DEV_Preparing", "language": "dev"},
+  {"keyName": "kitsPageStatusShipped", "text": "DEV_Shipped", "language": "dev"},
+  {"keyName": "kitsPageStatusReturned", "text": "DEV_Returned", "language": "dev"},
+  {"keyName": "kitsPageStatusCreated", "text": "DEV_Created", "language": "dev"},
+  {"keyName": "kitsPageStatusCollected", "text": "DEV_Collected", "language": "dev"},
+  {"keyName": "kitsPageNoKits", "text": "DEV_You do not have any sample collection kits at this time.", "language": "dev"},
+  {"keyName": "inPersonKitsTitle", "text": "DEV_Sample kit instructions", "language": "dev"},
+  {"keyName": "inPersonKitsDescription", "text": "DEV_If you are completing a sample collection kit in-person, please follow the instructions provided by a member of the study team. Any additional information that you may need, such as your unique participant identifier, will be provided below.", "language": "dev"},
```

Comment:
I would lean towards having these keys be "kitsInPerson..." just so they'll be grouped alphabetically with other kit stuff